README.md

```diff
@@ -58,15 +58,15 @@ This repo contains 3 main folders:
 
 <hr>
 
-*Cohort 3 [Europe/Africa]: (starting 2020-08-04) Tuesdays, 19:15 CET (CEST)*
-- 2020-08-04 Chapter 1 - Introduction [Novica Nakov](https://twitter.com/novicanakov) (probably :))
-- 2020-08-11 Chapter 2 - Names and values 
-- 2020-08-18 Chapter 3 - Vectors
+*Cohort 3 [Europe/Africa]: (started 2020-08-04) Tuesdays, 19:15 CET (CEST)*
+- 2020-08-04 Chapter 1 - Introduction [Novica Nakov](https://twitter.com/novicanakov) 
```

Comment:
I've been removing the entry for completed meetings from this level and adding an entry to Presentations and Recordings with a link to the slides + video etc :)

Reply:
We've been also collecting the slide RMD and HTMLs etc into the appropriate folders in the repo, so it would be great if you could upload those!

Reply:
Aha OK. 
We weren't using any slides for this one so nothing to add.

Reply:
Ah neat! Maybe just wait to do the video link along with the PR then? :)

Reply:
Should we also make a PR for putting in our contact info here or wait? (doing the chapter 2 presentation next week)

Reply:
@aleighbrown yes. if you want to have your contact info, you can add twitter, blog or whatever you prefer.

Reply:
@tanho63 Updated with video link.

Reply:
@aleighbrown yeah you’re welcome to make a PR to add your contact info! I’ll merge this in and then we should be good.